shared/fixes/bash/network_ipv6_disable_rpc.sh

```diff
@@ -0,0 +1,10 @@
+# platform = multi_platform_rhel
+
+# Drop 'tcp6' and 'udp6' entries from /etc/netconfig to prevent RPC
+# services for NFSv4 from attempting to start IPv6 network listeners
+declare -a IPV6_RPC_ENTRIES=("tcp6" "udp6")
+
+for rpc_entry in ${IPV6_RPC_ENTRIES[@]}
```

Comment:
It is a good habit to double quote @-expanded bash arrays `"${IPV6_RPC_ENTRIES[@]}"`. In this case, you don't need an array, a space-delimited string would be just fine.